src/test/java/core/basesyntax/CsvFileReaderTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.impl.CsvFileReader;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class CsvFileReaderTest {
+    private final String fromFilePath = "src/test/resources/test-database.csv";
+    private final Reader fileReader = new CsvFileReader();
```

Comment:
```suggestion
    private final Reader fileReader;
```
move to a method with annotation Before